ui/src/components/app_template.vue

```diff
@@ -50,6 +50,12 @@
   import the_footer from 'components/the_footer'
   import uploads_progress from 'components/uploads_progress'
 
+  has_files = (evt) ->
+    if evt.dataTransfer.types
```

Comment:
What's the guarantee that all events have `dataTransfer`?

Reply:
As far as I know, all drag-and-drop events have that prop. I could still add an extra check though just to make sure.